jmclient/jmclient/wallet-rpc-api.yaml

```diff
@@ -0,0 +1,618 @@
+          type: boolean
+        coinjoin_in_process:
+          type: boolean
+        wallet_name:
+          type: string
+          example: wallet.jmdat
+    ListUtxosResponse:
+      type: object
+      properties:
+        utxos: 
+          type: array
+          items: 
+            type: object
+            properties: 
+              utxo: 
+                type: string
+              address: 
+                type: string
+              value: 
+                type: integer
+              tries: 
+                type: integer
+              tries_remaining: 
+                type: integer
+              external: 
+                type: boolean
+              mixdepth: 
+                type: integer
+              confirmations: 
+                type: integer
+              frozen: 
+                type: boolean
+    WalletDisplayResponse:
+      type: object
+      required:
+        - walletname
+        - walletinfo
+      properties: 
+        walletname: 
+          type: string
+        walletinfo: 
+          type: object
+          required:
+            - wallet_name
+            - total_balance
+            - accounts
+          properties: 
+            wallet_name: 
+              type: string
+            total_balance: 
+              type: string
+            accounts: 
+              type: array
+              items: 
+                type: object
+                properties: 
+                  account: 
+                    type: string
+                  account_balance: 
+                    type: string
+                  branches: 
+                    type: array
+                    items: 
+                      type: object
+                      properties: 
+                        branch: 
+                          type: string
+                        balance: 
+                          type: string
+                        entries: 
+                          type: array
+                          items: 
+                            type: object
+                            properties: 
+                              hd_path: 
+                                type: string
+                              address: 
+                                type: string
+                              amount: 
+                                type: string
+                              labels: 
+                                type: string
+
+    CreateWalletResponse:
+      type: object
+      required:
+        - walletname
+        - already_loaded
+        - token
+        - seedphrase
+      properties:
+        walletname:
+          type: string
+          example: wallet.jmdat
+        already_loaded:
+          type: boolean
+          example: false
+        token:
+          type: string
+          format: byte
+        seedphrase:
+          type: string
+    UnlockWalletResponse:
+      type: object
+      required:
+        - walletname
+        - already_loaded
+        - token
```

Comment:
I think I might have lost some comments in a page reload but: since unlockwallet will only return an error now, if either this or another wallet is unlocked, after https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/996/commits/71c986b15b624378ff905aba55b3932e798e1ff9, and the `already_loaded` field is removed, it means that there is no longer a response that will not include the token.